src/js/components/Diagram/doc.js

```diff
@@ -64,6 +68,16 @@ export const themeDoc = {
     type: 'string | {dark: string, light: string}',
     defaultValue: 'accent-1',
   },
+  'global.animation': {
+    description: 'The animation configuration for Diagram.',
+    type: 'object',
+    defaultValue: `{
+      duration: '1s',
+      draw: {
+        duration: '0.1s',
```

Comment:
It is not clear how `duration` didders from `draw.duration`

Reply:
Looked into this today via a codesandbox, and it seems that you can specify a "catch-all" duration via `duration: "1s"` and for specific animations (in this case draw), you can also specify a specific duration for it as well.

Reply:
Top priority - clean up `global.animation.draw.duration`, this prop doesn't make sense to be placed on global if its only specific to Diagram. Move it to Diagram theme instead and account for it.